packages/flutter_tools/lib/src/commands/analyze_continuously.dart

```diff
@@ -122,220 +128,24 @@ class AnalyzeContinuously extends AnalyzeBase {
 
       final String files = '${analyzedPaths.length} ${pluralize('file', analyzedPaths.length)}';
       final String seconds = (analysisTimer.elapsedMilliseconds / 1000.0).toStringAsFixed(2);
-      printStatus('$errorsMessage • analyzed $files, $seconds seconds');
+      printStatus('$errorsMessage • analyzed $files in $seconds seconds');
 
       if (firstAnalysis && isBenchmarking) {
-        writeBenchmark(analysisTimer, issueCount, -1); // TODO(ianh): track members missing dartdocs instead of saying -1
+        // We don't want to return a failing exit code based on missing documentation.
+        issueCount -= undocumentedCount;
+
+        writeBenchmark(analysisTimer, issueCount, undocumentedCount);
         server.dispose().whenComplete(() { exit(issueCount > 0 ? 1 : 0); });
       }
 
       firstAnalysis = false;
     }
   }
 
-  bool _filterError(AnalysisError error) {
-    // TODO(devoncarew): Also filter the regex items from `analyzeOnce()`.
-
-    if (error.type == 'TODO')
-      return true;
-
-    return false;
-  }
-
   void _handleAnalysisErrors(FileAnalysisErrors fileErrors) {
-    fileErrors.errors.removeWhere(_filterError);
+    fileErrors.errors.removeWhere((AnalysisError error) => error.type == 'TODO');
```

Comment:
can't we turn these off in the options file?

Reply:
We now have the same options file for IDE and cli analysis. You do want to see these in the IDE (or use IDE level settings to hide them). They're not valuable on the cli; we filter them out from the dartanalyzer cli tool as well.